src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -917,8 +917,8 @@ private function _display_rows_hierarchical( $pages, $pagenum = 1, $per_page = 2
 	}
 
 	/**
-	 * Given a top level page ID, display the nested hierarchy of sub-pages
-	 * together with paging support
+	 * Displays the nested hierarchy of sub-pages
```

Comment:
Suggestion:
```
	 * Displays the nested hierarchy of sub-pages together with paging
	 * support, based on a top level page ID.
```

Reply:
Fixed in [e3f2e97](https://github.com/WordPress/wordpress-develop/pull/4552/commits/e3f2e9783fd2a232f4275a6abeea815c1b133a07).